nvtabular/io/dataset_collection.py

```diff
@@ -0,0 +1,102 @@
+import os
+from typing import Any, Optional
+
+import pandas as pd
+
+from nvtabular.io.dataset import LOG, Dataset
+from nvtabular.utils import Namespace
+
+
+class DatasetCollection(Namespace):
```

Comment:
`DatasetCollection` looks like it's specific to splits of a dataset, which makes the name seem more generic than the implementation. If splits are currently the only use case for this, maybe this class should be called something like `DatasetSplits`?